packages/manager/apps/pci-volume-backup/src/data/hooks/useVolumeBackups.tsx

```diff
@@ -0,0 +1,166 @@
+import { useState, useEffect, useMemo, useCallback } from 'react';
+import {
+  IcebergFetchParamsV6,
+  applyFilters,
+  FilterTypeCategories,
+  FilterComparator,
+  fetchIcebergV6,
+  Filter,
+} from '@ovh-ux/manager-core-api';
+import { Query, useQuery } from '@tanstack/react-query';
+import {
+  DatagridColumn,
+  useColumnFilters,
+  ColumnSort,
+} from '@ovh-ux/manager-react-components';
+
+export type ApiData<T> = {
+  data: T;
+};
+
+export interface ResourcesV6Hook<T> {
+  queryKey: string[];
+  queryFn?: () => Promise<ApiData<T[]>>;
+  refetchInterval?: (query: Query<ApiData<T[]>>) => number | false;
+  columns: DatagridColumn<T>[];
+  defaultSorting?: ColumnSort;
+}
+
+/* eslint-disable  @typescript-eslint/no-explicit-any */
+function sortColumn(
+  type: FilterTypeCategories,
+  a: string,
+  b: string,
+  desc: boolean,
+) {
+  if (!a || !b) return -1;
+  switch (type) {
+    case FilterTypeCategories.Numeric:
```

Comment:
Why not use the value directly instead of stringify + parse ?

Reply:
Same, it will be removed to use `useResourcesV6` instead

Reply:
But to answer your question, I think it's because the type of the value is `string | string[]`. It's a code taken from MRC or core API

Reply:
Done